src/Compile/Options.hs

```diff
@@ -853,7 +854,10 @@ processInitialOptions flags0 opts
               (localDir,localLibDir,localShareDir,localBinDir)
                   <- getKokaDirs (localLibDir flags) (localShareDir flags) buildDir
 
-              normalizedIncludes <- mapM realPath ("." : (localShareDir ++ "/lib") : includePath flags)
+              pathsFromEnv <- fmap splitSearchPath (getEnvVar "KOKA_PATH")
```

Comment:
It looks like we already have a utility for splitting delimited paths used elsewhere in this file: `undelimPaths`. Not sure if Daan would want to avoid the extra dependency, and use this, or if we should use a version that might handle slightly more edge cases than we do.

Reply:
Oh yep happy to use that, I wasn't aware of it. GHC reported that `filepath` was a hidden dependency which I assumed meant it was already required transitively, but there's a chance it was only used for tests previously.

Reply:
Hmm, yes, that might be the case.